tool/plugin/lib/plugin.dart

```diff
@@ -388,7 +388,10 @@ class GradleBuildCommand extends ProductCommand {
       }
 
       separator('Built artifact');
-      log(releasesFilePath(spec));
+      final releaseFilePath = releasesFilePath(spec);
+      final file = File(releaseFilePath);
+      final fileSize = file.lengthSync() / 1000000;
```

Comment:
nit: this can be condensed: `final file = File(releaseFilePath).lengthSync() / 1000000`